### PR TITLE
New property in Scene that contains all instances of canvas2d

### PR DESCRIPTION
**Pull request by Temechon**

### CODE REVIEW

File: src/babylon.scene.ts [hunk 1]
```diff
@@ -339,6 +339,9 @@
         */
         public meshes = new Array<AbstractMesh>();
 
+        // Contain all canvas2D used in the scene
+        public canvas2ds = new Array<Canvas2D>();
+
```
Comment (nockawa) [inline src/babylon.scene.ts]: Please, no Canvas2D stuff in Scene.ts.
Instead you should rely on a public get Canvas(): Array<Canvas2D> in Canvas2D class with a private static class behind.

Comment (deltakosh) [inline src/babylon.scene.ts]: Yep Canvas2d is not in core so cannot have a strong reference,

File: dist/preview release/what's new.md [hunk 2]
```diff
@@ -26,6 +26,7 @@
  - Text2D super sampling to enhance quality in World Space Canvas
  - World Space Canvas is now rendering in an adaptive way for its resolution to fit the on screen projected one to achieve a good rendering quality
  - Transparent Primitives are now drawn with Instanced Array when supported
+ - New property in Scene (canvas2ds) that contains all instances of canvas2d in this scene [Temechon](https://github.com/Temechon)
```
Comment (nockawa) [inline dist/preview release/what's new.md]: not anymore! ;)

Comment (nockawa) [inline dist/preview release/what's new.md]: Update the what's new and it's good to go.

Comment (Temechon) [inline dist/preview release/what's new.md]: Already done !

**Pull request merged**